python/cudf/cudf/core/column/categorical.py

```diff
@@ -656,6 +688,25 @@ def _mimic_inplace(self, other_col, inplace=False):
         return out
 
 
+def _create_empty_categorical_column(categorical_column, dtype):
+    from cudf.utils import utils
```

Comment:
Don't do imports within functions like this, it actually turns out to be expensive because of handling lists, not exactly sure why...

Just use `cudf.utils.utils` within the function.